core/src/serve_repair.rs

```diff
@@ -468,7 +469,7 @@ impl ServeRepair {
             }
         }
 
-        stats.dropped_requests += dropped_requests;
```

Comment:
hmmm, now i am wondering if we are unnecessary dropping requests here.

Reply:
yeah, now we'll find out how many are being dropped here.